examples/high_frequency/emc/busbar.py

```diff
@@ -117,18 +117,66 @@
 print(q3d.net_sources("Bar2"))
 print(q3d.net_sources("Bar3"))
 
+
+# ## Create Matrix Reduction Operations
+#
+# Series of Bar1 and Bar2
+
+mr_series = q3d.insert_reduced_matrix(
+    operation_name="JoinSeries",
+    assignment=["Sink1", "Source2"],
+    reduced_matrix="MR_1_Series",
+    new_net_name="Series1",
+)
+
+# Add Parallel with Bar3
+
+mr_series.add_operation(
+    operation_type="JoinParallel",
+    source_names=["Source1", "Source3"],
+    new_net_name="SeriesPar",
+    new_source_name="src_par",
+    new_sink_name="snk_par",
+)
+
+# Series of Bar1 and Bar2
+
+mr_series2 = q3d.insert_reduced_matrix(
+    operation_name="JoinSeries",
+    assignment=["Sink1", "Source2"],
+    reduced_matrix="MR_2_Series",
+    new_net_name="Series2",
+)
+
+# Add Series with Bar3
+
+mr_series2.add_operation(
+    operation_type="JoinSeries",
+    source_names=["Sink3", "Source1"],
+    new_net_name="MR_2_Series1",
+)
+
 # Create the solution setup and define the frequency range for the solution.
```

Comment:
@tizianrot maybe here you can create a section for the creation of the setup and analysis differentiating from the previous section "Create Matrix Reduction Operations"